frontend/src/api/prometheus/distributedWorkloads.ts

```
@@ -133,9 +134,20 @@ export const DEFAULT_DW_PROJECT_CURRENT_METRICS: DWProjectCurrentMetrics = {
 };
 
+const getLWSMetricSubquery = (namespace: string, metricName: string): string =>
+  `sum by(owner_name, owner_kind) (label_replace(label_replace(kube_pod_labels{label_leaderworkerset_sigs_k8s_io_name!="", namespace="${namespace}"} * on (namespace, pod) group_right(label_leaderworkerset_sigs_k8s_io_name) ${metricName}, "owner_name", "$1", "label_leaderworkerset_sigs_k8s_io_name", "(.+)"), "owner_kind", "LeaderWorkerSet", "", ""))`;
+
 const getDWProjectCurrentMetricsQueries = (
   namespace: string,
 ): Record<DWProjectCurrentMetricType, string> => ({
+  cpuCoresUsedByWorkloadOwner: `namespace=${namespace}&query=sum by(owner_name, owner_kind) ((kube_pod_owner{owner_kind=~"RayCluster|Job|StatefulSet", namespace="${namespace}"} unless on (namespace, pod) kube_pod_labels{label_leaderworkerset_sigs_k8s_io_name!="", namespace="${namespace}"}) * on (namespace, pod) group_right(owner_name, owner_kind) node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate) or ${getLWSMetricSubquery(
+    namespace,
+    'node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate',
+  )}`,
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)